course/src/isometry.cpp

```diff
@@ -10,9 +10,136 @@
 namespace ekumen {
 namespace math {
 
-/*
- * your code here
- */
+namespace {
+
+Vector3 translation_vector_operation(const Matrix3& matrix,
+                                     const Vector3& first_vector,
+                                     const Vector3& second_vector) {
+  return matrix * first_vector + second_vector;
+}
+
+}  // namespace
+
+
+Isometry Isometry::fromTranslation(const Vector3& r_vector) {
+  return Isometry{r_vector, Matrix3::kIdentity};
+}
+
+Vector3 Isometry::translation() const {
+  return  Vector3{translation_vector};
```

Comment:
nit: 
```suggestion
  return  translation_vector;
```
otherwise, you are creating an object `Vector3{translation_vector}` before passing it. And `translation_vector` is already a Vector3.

Reply:
Addressed.